setup.cfg

```diff
@@ -1,2 +1,11 @@
 [bdist_wheel]
-python_tag = py3
\ No newline at end of file
+universal = 1
```

Comment:
This is not a universal wheel because it does not work with Python 2.

From https://wheel.readthedocs.io/en/stable/user_guide.html#building-wheels:

> If your project contains no C extensions and is expected to work on both Python 2 and 3, you will want to tell wheel to produce universal wheels by adding ...

According to https://www.python.org/dev/peps/pep-0425/#id11, the `py3` tag seems correct.